guacamole/src/main/webapp/translations/fr.json

```diff
@@ -158,7 +173,10 @@
         "SECTION_HEADER_RECENT_CONNECTIONS" : "Connexions récentes"
 
     },
-
+    
+    "LIST":{
+         "TEXT_ANONYMOUS_USER" : "Anonyme"
```

Comment:
Style here should match other files - there should be blank lines before and after this line.